src/audio/audio.cpp

```diff
@@ -87,23 +87,23 @@ std::unique_ptr<Response> PipeAudio::Emit(const std::string &path,
 
 	std::string value;
 
-	if (path == "/control/state") {
```

Comment:
`/control/state` will need to be a thing for compliance, but it'll just return `running` every time